cmd/ftl/cmd_schema_import.go

```diff
@@ -0,0 +1,220 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/TBD54566975/ftl/backend/common/exec"
+	"github.com/TBD54566975/ftl/backend/common/log"
+	"github.com/TBD54566975/ftl/backend/common/slices"
+	"github.com/tmc/langchaingo/llms"
+	"github.com/tmc/langchaingo/llms/ollama"
+)
+
+type schemaImportCmd struct {
+	OllamaPort int             `help:"Port to use for ollama." default:"11434"`
+	Go         importGoCmd     `cmd:"" help:"Import types for an FTL Go module."`
+	Kotlin     importKotlinCmd `cmd:"" help:"Import types for an FTL Kotlin module."`
+}
+
+type importGoCmd struct {
+	Dir string `arg:"" required:"" help:"Directory to import from."`
+}
+
+type importKotlinCmd struct {
+	Dir string `arg:"" required:"" help:"Directory to import from."`
+}
+
+const ollamaContainerName = "ftl-ollama-1"
+const ollamaVolume = "ollama:/root/.ollama"
+const ollamaModel = "llama2"
+
+func (i *importKotlinCmd) getPrompt() (string, error) {
+	input, err := createInputString(i.Dir)
+	if err != nil {
+		return "", err
+	}
+
+	return fmt.Sprintf("I will provide you with an input string containing the contents of a file which must be translated into Kotlin. "+
```

Comment:
would love some input on these prompts!

per the disclaimer logging below (`The import schema command relies on the llama2 AI chat model to translate schemas from other languages into FTL compliant objects in the specified language. Output may vary and results should be inspected for correctness. It is suggested that if the results are not satisfactory, you try again.`), I had mixed results when testing this locally. the output was usually mostly correct but perhaps people have other ideas on how to query more effectively.

Reply:
Can we make an allowance to plug in GPT? maybe it's via exposing the engineer's API token in a .gpt file or something?
Its codegen is much better afaik

Reply:
Yeah agreed, we discussed exactly that on Slack 👍